packages/flutter_tools/test/integration/lifetime_test.dart

```diff
@@ -0,0 +1,54 @@
+// Copyright 2018 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+
+import 'package:file/file.dart';
+import 'package:flutter_tools/src/base/file_system.dart';
+import 'package:test/test.dart';
+
+import 'test_data/basic_project.dart';
+import 'test_driver.dart';
+
+BasicProject _project = new BasicProject();
+FlutterTestDriver _flutter;
+
+const Duration requiredLifespan = Duration(seconds: 5);
```

Comment:
Perhaps add comment how this constant is chosen (e.g. that it is arbitrary but seems good enough)

Reply:
Added a comment.